packages/google-pay-integration/src/google-pay-paypal-commerce/google-pay-paypal-commerce-gateway.ts

```diff
@@ -18,34 +20,42 @@ import { GooglePayConfig } from './types';
 
 export default class GooglePayPaypalCommerceGateway extends GooglePayGateway {
     private googlepayConfig?: GooglePayConfig;
-    private paymentMethod?: PaymentMethod<GooglePayPayPalCommerceInitializationData>;
+    private paymentMethod?: PaymentMethod<GooglePayInitializationData>;
```

Comment:
Do we really need to create class variable for payment method? I can see, that it used only in one method

Reply:
Yes. For correctly initialization of paypal-sdk with necessary parameters we get from payment method